src/anml/parameter/parameter.py

```diff
@@ -169,6 +174,8 @@ def __post_init__(self):
             raise ValueError('Lower bound cannot be greater than upper bound.')
         if self.order < 0:
             raise ValueError('Order of derivative must be nonnegative.')
+        if self.grid_size < 1:
```

Comment:
Seems like when `grid_size` is `None` which is default and legal value, this part will definitely fail.

Reply:
oops, fixed that

Reply:
great! Thanks!